jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -73,6 +73,15 @@ def should_export_dbias(bias_shape, query_shape, layout) -> bool:
     _, _, q_N, _ = query_shape
   return b_B == 1 and b_N == q_N
 
+def get_large_negative_number(dtype):
+  # temp WAR as cuDNN has a bug for subtraction between two large negative value
```

Comment:
When can this workaround be removed? Is there a bug you can link to here?

Reply:
It is still not fixed in cuDNN AFAIK. So when cuDNN subtract two large negative number (softmax for example) will result in nan. There is an internal nvbug but nothing I can link here.